pkg/agent/clean/binding.go

```diff
@@ -171,36 +173,40 @@ func (bc *bindingsCleanup) cleanObjectDuplicates(bindingType string, newLabel bo
 	for _, meta := range objMetas {
 		labels := createLabelSelectors(newLabel, meta, bindingType)
 		for _, label := range labels {
+			logrus.Debugf("%v %v label:%v, Checking CRB/RB duplicates...", bindingUpper, meta.Name, label)
```

Comment:
Can this log be switch around along with the one below 
```suggestion
			logrus.Debugf("Checking CRB/RB duplicates for: %v %v label:%v", bindingUpper, meta.Name, label)
```

Reply:
I was going for consistency with the existing log message (well, with added comma to help separate things).
```
logrus.Infof("%v %v label:%v Duplicates: CRB:%v RB:%v", bindingUpper, meta.Name, label, CRBduplicates, RBDupes)
```

So we can either keep this one as is or change all 3 log messages . I'm fine either way, let me know what you would prefer.

P.S. On a side note, it would be great if we did structured logging.

Reply:
Switch the calls to use logrus.WithFields if you want some structure to these, which would probably help. The original logs were mainly for the job and consumption through there so I would prefer them to be easier to read now that this runs inside rancher as well. So log message first then data is my preference.

Reply:
That would be odd to have a single place where we're using structured logging, so I've switched the messages as you suggested with the actual message first followed by the data.